src/GraphSDKGenerator/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CommandLine;
+using Microsoft.Graph.ODataTemplateWriter.TemplateProcessor;
+using NLog;
+using NLog.Config;
+using NLog.Targets;
+using Vipr.Core;
+using Vipr.Core.CodeModel;
+using Vipr.Reader.OData.v4;
+
+namespace GraphSDKGenerator
+{
+    class Program
+    {
+        internal static Logger Logger => LogManager.GetLogger("GraphSDKGenerator");
+
+        static void Main(string[] args)
+        {
+            Parser.Default.ParseArguments<Options>(args)
+                .WithParsed(opts => GenerateSDK(opts))
+                .WithNotParsed((errs) => HandleError(errs));
+        }
+
+        private static void GenerateSDK(Options opts)
+        {
+            var stopwatch = new Stopwatch();
+            stopwatch.Start();
+
+            SetupLogging(opts.Verbosity);
+
+            var edmxContents = MetadataResolver.GetMetadata(opts.Metadata);
```

Comment:
Change edmxContents to csdlContents per:

![image](https://user-images.githubusercontent.com/8527305/44759543-a65fc500-aaef-11e8-8f1c-23059497c2a1.png)